But `ext/POSIX/t/time.t` still fails:
```
ext/POSIX/t/termios .............................................. ok
ext/POSIX/t/time ................................................. Cannot setlocale(LC_CTYPE) back to orig: No such file or directory at t/time.t line 89.
# Looks like your test exited with 2 just after 14.
FAILED--expected 19 tests, saw 14
ext/POSIX/t/unimplemented ........................................ ok
[...]
Failed 1 test out of 2452, 99.96% okay.
	../ext/POSIX/t/time.t
### Since not all tests were successful, you may want to run some of
### them individually and examine any diagnostic messages they produce.
### See the INSTALL document's section on "make test".
### You have a good chance to get more information by running
###   ./perl harness
### in the 't' directory since most (>=80%) of the tests succeeded.
### You may have to set your dynamic library search path,
### DYLD_LIBRARY_PATH, to point to the build directory:
###   setenv DYLD_LIBRARY_PATH `pwd`; cd t; ./perl harness
###   DYLD_LIBRARY_PATH=`pwd`; export DYLD_LIBRARY_PATH; cd t; ./perl harness
###   export DYLD_LIBRARY_PATH=`pwd`; cd t; ./perl harness
### for csh-style shells, like tcsh; or for traditional/modern
### Bourne-style shells, like bash, ksh, and zsh, respectively.
Elapsed: 2183 sec
u=13.09  s=8.11  cu=713.08  cs=109.96  scripts=2452  tests=1184898
make: *** [test] Error 1
```
Maybe this failure should be reported in a separate issue?

--- a/ext/DynaLoader/DynaLoader_pm.PL
+++ b/ext/DynaLoader/DynaLoader_pm.PL
@@ -88,7 +88,7 @@ package DynaLoader;
 # Tim.Bunce@ig.co.uk, August 1994
 
 BEGIN {
-    $VERSION = '1.49';
+    $VERSION = '1.50';
 }
 
 EOT
@@ -494,12 +494,20 @@ sub dl_findfile {
             foreach $name (@names) {
 		my($file) = "$dir$dirsep$name";
                 print STDERR " checking in $dir for $name\n" if $dl_debug;
-		$file = ($do_expand) ? dl_expandspec($file) : (-f $file && $file);
-		#$file = _check_file($file);
-		if ($file) {
+		if ($do_expand && ($file = dl_expandspec($file))) {
+                    push @found, $file;
+                    next arg; # no need to look any further
+		}
+		elsif (-f $file) {
                     push(@found, $file);
                     next arg; # no need to look any further
                 }
+		<<$^O-eq-darwin>>
+		elsif (dl_load_file($file, 0)) {
+                    push @found, $file;
+                    next arg; # no need to look any further
+		}
+		<</$^O-eq-darwin>>
             }
         }
     }

